community.html

```diff
@@ -21,9 +21,13 @@ <h1 class="alt-h2 text-center">Join the Community</h1>
       Contribute on GitHub
     </a>
 
-    <a class="btn btn-outline d-inline-block mb-1 mb-md-0" href="https://twitter.com/ProbotTheRobot">
+    <a class="btn btn-outline d-inline-block mb-1 mb-md-0 mr-0 mr-md-3" href="https://twitter.com/ProbotTheRobot">
       Follow on Twitter
     </a>
+
+    <a class="btn btn-outline d-inline-block mb-1" href="https://stackoverflow.com/questions/tagged/probot">
+      Probot on Stack Overflow
```

Comment:
The other ones are "Chat on…", "Contribute on…", "Follow on…"

What about "Ask on Stack Overflow"?

Reply:
yeh that works.